src/Files.App/Interacts/BaseLayoutCommandsViewModel.cs

```diff
@@ -174,6 +176,10 @@ private void InitializeCommands()
 
 		public ICommand CompressIntoArchiveCommand { get; private set; }
 
+		public ICommand CompressIntoZipCommand { get; private set; }
+
+		public ICommand CompressIntoSevenZipCommand { get; private set; }
+
```

Comment:
Can we merge them into `CompressIntoArchiveCommand `?

Reply:
A command should never use a parameter. In particular this case which concerns very different behaviors. Using commands without parameters is needed to customize hotkeys (another feature in progress).